wkt.build.js

```diff
@@ -297,6 +297,10 @@ function d2r(input) {
 }
 
 function cleanWKT(wkt) {
+  if (wkt.AUTHORITY) {
+    var authority = Object.keys(wkt.AUTHORITY)[0];
+    wkt.title = `${authority}:${wkt.AUTHORITY[authority]}`;
+  }
```

Comment:
This change is unrelated, and `wkt.build.js` should not be manually modified, because it is auto-built.

Reply:
I need to update the README and add that file to .gitignore. Sorry that the instructions are currently not clear about that.